app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCTableView.scala

```diff
@@ -133,6 +135,18 @@ class DLCTableView(model: DLCPaneModel) {
         }
       }
 
+      val viewOnExplorer: MenuItem = new MenuItem("View on Oracle Explorer") {
+        onAction = _ => {
+          val status = selectionModel.value.getSelectedItem
+          val primaryOracle =
+            status.oracleInfo.singleOracleInfos.head.announcement
+          val baseUrl = ExplorerEnv.Production.siteUrl
```

Comment:
Is there any way to pattern match on the network setting? 

Oracles are network agnostic, but it seems we should probably point to the test explorer when users are on testnet3?

Reply:
This might be a nice feature for @user411 to implement. It is pretty small in scope, should be easy to view on his local wallet too. 

If we think this is a good idea, we should merge this PR, and file an issue for him to implement the testnet3 feature. Thoughts?